src/components/Chart/Maps.docs.md

```diff
@@ -774,40 +901,43 @@ JU,0.257
       "choropleth": true,
       "numberFormat": ".0%",
       "thresholds": [0.3,0.4,0.5,0.6,0.7],
+      "colorLegendLabels": ["unter 30%", "", "", "", "", "über 70%"],
```

Comment:
not quite happy with that API yet, that an empty string triggers the default label is not intuitive and skipping is not possible, let's discuss if we should automatically do nicer labels for edge values or a more explicit API

Reply:
yeah I was a bit on the fence regarding that one. my other idea would be to be able to do something like `["unter 30%", "auto", "auto", "auto", "auto", "über 70%"]`
would it sit better with you?

Reply:
`thresholdConfig`

```
[
   {"label": "Weniger als 30%"},
   {minValue: 0.3, "label": "30% bis 40%"},
   {minValue: 0.4},
   {minValue: 0.5, "label": "Über 50%"}
]
```